src/test/java/org/folio/rest/api/InstanceStorageTest.java

```diff
@@ -448,7 +449,49 @@ public void canGetAllInstances()
     assertThat(secondInstance.getJsonArray("identifiers"),
       hasItem(identifierMatches(UUID_ASIN.toString(), "B01D1PLMDO")));
   }
+  @Test
+  public void canGetWithAllIndex()
+    throws MalformedURLException,
+    InterruptedException,
+    ExecutionException,
+    TimeoutException {
+
+    UUID firstInstanceId = UUID.randomUUID();
+
+    JsonObject firstInstanceToCreate = smallAngryPlanet(firstInstanceId);
 
+    createInstance(firstInstanceToCreate);
+
+    UUID secondInstanceId = UUID.randomUUID();
+
+    JsonObject secondInstanceToCreate = nod(secondInstanceId);
+
+    createInstance(secondInstanceToCreate);
+
+    CompletableFuture<Response> getCompleted = new CompletableFuture<>();
+
+    client.get(instancesStorageUrl("?query=keyword%3D%22Long%20Way%20to%20a%20Small%20Angry%20Planet%20Chambers%2C%20Becky%209781473619777%22"), StorageTestSuite.TENANT_ID,
+        ResponseHandler.json(getCompleted));
+
+    Response response = getCompleted.get(5, TimeUnit.SECONDS);
+
+    JsonObject responseBody = response.getJson();
+
+    JsonArray allInstances = responseBody.getJsonArray("instances");
+
+    assertThat(allInstances.size(), is(1));
+    assertThat(responseBody.getInteger("totalRecords"), is(1));
+
+    JsonObject firstInstance = allInstances.getJsonObject(0);
+
+
+    assertThat(firstInstance.getString("id"), is(firstInstanceId.toString()));
+    assertThat(firstInstance.getString("title"), is("Long Way to a Small Angry Planet"));
+
+    assertThat(firstInstance.getJsonArray("identifiers").size(), is(1));
+    assertThat(firstInstance.getJsonArray("identifiers"),
+      hasItem(identifierMatches(UUID_ISBN.toString(), "9781473619777")));
+  }
```

Comment:
Can you add a blank line between methods?

Reply:
removed these lines due to not being needed

Reply:
lines removed